numpyro/util.py

```diff
@@ -19,14 +19,14 @@
 _DISABLE_CONTROL_FLOW_PRIM = False
 
 
-def set_rng_seed(rng_seed):
+def set_rng_key_seed(rng_key_seed):
```

Comment:
This utility does not relevant to `rng_key` so it is better to keep the old version.

Reply:
good catch, will fix.